ftw/book/tests/test_listingblock_latex.py

```diff
@@ -41,13 +41,13 @@ def test_adds_width_to_columns(self):
                 '<col />',
                 '</colgroup></table>'))
 
-        expected = ''.join((
-                '<table><colgroup>',
+        expected = '\n'.join((
+                '<div><table><colgroup>',
                 '<col class="col-modified" width="12%">',
                 '<col class="col-Creator" width="15%">',
                 '<col class="col-getObjSize" width="10%">',
                 '<col width="63%">',
-                '</colgroup></table>'))
+                '</colgroup></table></div>'))
```

Comment:
Fixed in https://github.com/4teamwork/ftw.book/pull/88 ; please rebase and :scissors: